experimental/c-cpp/manager.py

```diff
@@ -1256,6 +1375,25 @@ def get_heuristics_to_use() -> List[Type[FuzzHeuristicGeneratorBase]]:
   return heuristics_to_apply
 
 
+def get_all_test_scripts(target_source_path) -> List[Test]:
+  """Returns a list of the test files in the target source path."""
+  all_files = get_all_files_in_path(target_source_path)
+  all_tests = []
+  for file in all_files:
+    split_path = file.split('/')
+    is_test = any(['test' in path for path in split_path])
+    if not is_test:
+      continue
+    if not file.endswith('.c') and not file.endswith('.cpp'):
+      continue
```

Comment:
Would this miss files like `.cc` or `.cxx`?
Maybe using [`is_c_file(file)`](https://github.com/google/oss-fuzz-gen/blob/4e3798ef7a6af994d7e09094cd399876e701d1a2/experiment/benchmark.py#L225) and [`is_cpp_file(file)`](https://github.com/google/oss-fuzz-gen/blob/4e3798ef7a6af994d7e09094cd399876e701d1a2/experiment/benchmark.py#L230)?

Reply:
Used the array! We're not copying the core OFG in so we can't used the module directly (would also cause issues with the Python version regarding the typing)

Reply:
Ah, the inconsistency in the Python version is annoying. But I guess we have to bear with it as there are more important changes.
The array should suffice.